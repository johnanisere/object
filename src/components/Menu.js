import React from "react"
import Menu from "../blocks/menu"
import { FaExpand, FaEllipsisH, FaRegHandPaper, FaSistrix } from "react-icons/fa";

const App = () => <Menu>
    <span className="active">
        <FaExpand />
    </span>
    <span className="more">
        <FaEllipsisH />
    </span>
    <span>
        <FaSistrix />
    </span>
    <span>
        <FaRegHandPaper />
    </span>

</Menu>

export default App