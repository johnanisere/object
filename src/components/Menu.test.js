
import React from "react";
import { shallow } from "enzyme";

import Menu from "./Menu"
import MenuContainer from "../blocks/menu"
import { FaExpand, FaEllipsisH, FaRegHandPaper, FaSistrix } from "react-icons/fa";

it("Menu component exists", () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper.exists()).toBe(true);
});

it("Menu component renders", () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper).toMatchSnapshot();
});

it(`Menu should contain "MenuContainer",  "span","" `, () => {
    const wrapper = shallow(<Menu />);

    expect(wrapper.find(FaExpand)).toHaveLength(1);
    expect(wrapper.find(FaSistrix)).toHaveLength(1);
    expect(wrapper.find(FaEllipsisH)).toHaveLength(1);
    expect(wrapper.find(MenuContainer)).toHaveLength(1);
    expect(wrapper.find(FaRegHandPaper)).toHaveLength(1);
});