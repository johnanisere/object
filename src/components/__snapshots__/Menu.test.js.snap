// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component renders 1`] = `ShallowWrapper {}`;
